src/app/@ansyn/ansyn/app-effects/effects/cases/select-case.app.effects.ts

```diff
@@ -39,6 +40,20 @@ export class SelectCaseAppEffects {
 		.ofType<SelectCaseAction>(CasesActionTypes.SELECT_CASE)
 		.mergeMap(({ payload }: SelectCaseAction) => this.selectCaseActions(payload));
 
+	@Effect()
+	setSpecialObjectsFromSelectedCase$: Observable<any> = this.actions$
```

Comment:
remove setSpecialObjectsFromSelectedCase$ add functionality to selectCase$

Reply:
done